site/student-lectures/cultural-algorithms/cultural-algorithms.rst

```diff
@@ -0,0 +1,255 @@
+*******************
+Cultural Algorithms
+*******************
+
+Introduction
+============
+* It is a branch of evolutionary computation
+* Unlike genetic algorithms, cultural algorithms have not only **population space** but also **belief space**
+* Here shows the components of cultural algorithms and its structure [#]_:
+
+.. figure:: ca_components.png
+    :align: center
```

Comment:
Add captions to the figures. I see you explain it below, but still add captions.